src/migrations/1720703882072-add-address-to-store.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddAddressToStore1720703882072 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "store" ADD COLUMN "address" character varying NULL`
```

Comment:
for now, Address is a string which can be parsed to JSON object.
later, we might make it an Entity